app/views/recipes/new.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Recipes#new</h1>
+<p>Find me in app/views/recipes/new.html.erb</p>
```

Comment:
- [ ] [OPTIONAL] It will be better to add a form to create a recipe to make all app functionality works because in your app you can create recipes only from the rails console.

Reply:
Yes, I planned to implement it outside program hours. It's not in the project requirement. Thanks for mentioning it.
@omarsalem7